web/themes/joinup/joinup_theme.theme

```diff
@@ -766,18 +766,10 @@ function joinup_theme_preprocess_field__comment(&$variables) {
   // Get the comment status from the entity itself.
   $comments_status = $variables['element']['#items']->status;
 
-  // On closed collections community content, anonymous users are not allowed to
-  // comment. Show a message instead.
+  // Anonymous users are not allowed to post comments.
   /** @var \Drupal\rdf_entity\RdfInterface $parent */
-  $parent = \Drupal::service('joinup_core.relations_manager')->getParent($entity);
   $user = \Drupal::currentUser();
-  if (
-    $user->isAnonymous() &&
-    $parent &&
-    $parent->getEntityTypeId() === 'rdf_entity' &&
-    $parent->bundle() === 'collection' &&
-    $parent->get('field_ar_closed')->value
-  ) {
+  if ($user->isAnonymous()) {
     $comments_status = 'closed_collection';
```

Comment:
This comment status is not relevant any more, let's change this to `authentication_required`.